ui/imports/assets/images/status-logo-light.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="256" height="98" viewBox="0 0 256 98" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think we should add these to status-go assets in general , under the icons folder? I am not sure if we have decided on this or not yet though.

Reply:
Do you mean StatusQ?

Reply:
Yas I mean StatusQ